packages/orbit-components/src/Tabs/__tests__/index.test.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Tabs, { TabList, Tab, TabPanels, TabPanel } from "..";
+
+describe("Tabs", () => {
```

Comment:
We should assert that the correct TabPanel is in the document on the different interactions. Since the component is responsible for handling that.